spring-cloud-gcp-samples/spring-cloud-gcp-data-datastore-basic-sample/src/test/java/com/example/DatastoreBookshelfExampleTests.java

```diff
@@ -64,6 +77,18 @@ public static void checkToRun() {
 				System.getProperty("it.datastore"), is("true"));
 	}
 
+	@Test
+	public void testSerializedPage() {
+		Book book = new Book("Book1", "Author1", 2019);
+		book.id = 12345678L;
+		this.bookRepository.save(book);
+		Awaitility.await().atMost(15, TimeUnit.SECONDS)
+				.until(() -> this.bookRepository.count() == 1);
+		String responseBody = sendRequest("/allbooksserialized", null, HttpMethod.GET);
+		assertThat(responseBody).contains("content\":[{\"id\":12345678}],\"pageable\":");
+		assertThat(responseBody).contains("\"urlSafeCursor\":");
```

Comment:
Can you check that it has a value using regex?

Reply:
added